src/main/java/se/sundsvall/seabloader/service/InvoiceService.java

```diff
@@ -29,4 +43,41 @@ public void create(final byte[] content) {
 
 		invoiceRepository.save(invoiceEntity);
 	}
+
+	public void exportInvoices() {
+		final var BATCH_SIZE = 10;
+		LOGGER.info("Exporting invoices to Invoice-cache");
+
+		final var invoiceIdsToSend = invoiceRepository.findInvoiceIdsByStatusIn(STATUSES_TO_SEND);
+
+		if (invoiceIdsToSend.isEmpty()) {
+			LOGGER.info("No invoices found with status {}", STATUSES_TO_SEND);
+			return;
+		}
+
+		LOGGER.info("Found {} invoices with status {}", invoiceIdsToSend.size(), STATUSES_TO_SEND);
+
+		new ArrayList<>(ListUtils
```

Comment:
What is the point of sending this 10 by 10?
Why not just loop through all invoices (i.e invoiceIdsToSend) and send them 1 by 1?